lab-kyle/test/main-test.js

```diff
@@ -0,0 +1,13 @@
+'use strict';
```

Comment:
This looks good and is the Travis CI failure culprit.  I suspect the change I've commented on below may resolve the issue.